canonical-web-and-design/ubuntu.com
# Updates cube terms of service

## Done

- Updates terms of service at /legal/terms-and-policies/cube-terms

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/legal/terms-and-policies/cube-terms
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare against [copy-doc](https://docs.google.com/document/d/1vRtCO-FDMC8rqthi4sgI1Fn3ofT6zqcCpjHbOyiyhjY/edit#)


## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4278
